tests/example_dags/test_example_databricks_notebook.py

```diff
@@ -56,10 +57,6 @@
             {"pypi": {"package": "Faker"}},
         ],
         source="WORKSPACE",
-        job_cluster_key="random_cluster_key",
+        job_cluster_key=str(uuid.uuid4()),
```

Comment:
🤔 didn't this approach cause us some trouble before?

Reply:
It caused us trouble for the DAG with the task group context. And I just check, this param is actually unused for launching a one-time job. So, I removed it from this example DAG.